test/plots/barPlotTests.ts

```diff
@@ -634,32 +634,18 @@ describe("Plots", () => {
         barPlot.renderTo(svg);
       });
 
-      it("hides labels properly on the right", () => {
```

Comment:
What happened to this test? Previously, there was functionality that hid labels if they would be cut off by the edges of the `Plot` area to avoid confusion (consider a case where the last few digits of "100000" are cut off), which seems to have gone missing.